llvm/lib/SYCL/KernelPropGen.cpp

```diff
@@ -138,72 +140,92 @@ struct KernelPropGen : public ModulePass {
                                .str() +
                            ' ';
               }
-              AddExtraArgsToCallers(CB->getFunction(), ArgsStr);
+              addExtraArgsToCallers(CB->getFunction(), ArgsStr);
             }
           }
       }
     }
   }
 
-  void GenerateVPPPropertyFile(Module &M, llvm::raw_fd_ostream &O) {
-    CollectExtraArgs(M);
+  void generateBundleSE(Argument &Arg,
```

Comment:
Add some comments, at least with a generic comment explaining what is done at high level